xds/internal/balancer/cdsbalancer/cdsbalancer_test.go

```diff
@@ -421,10 +423,11 @@ func (s) TestConfigurationUpdate_MissingXdsClient(t *testing.T) {
 	r.InitialState(resolver.State{ServiceConfig: scpr})
 
 	// Create a ClientConn with the above manual resolver.
-	cc, err := grpc.Dial(r.Scheme()+":///test.service", grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithResolvers(r))
+	cc, err := grpc.NewClient(r.Scheme()+":///test.service", grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithResolvers(r))
 	if err != nil {
-		t.Fatalf("Failed to dial: %v", err)
+		t.Fatalf("Failed to create gRPC client: %v", err)
```

Comment:
"Failed to create new client". Lets be consistent with language

Reply:
done please check